egs2/wsj0_2mix_spatialized/enh1/README.md

```diff
@@ -1,5 +1,30 @@
 <!-- Generated by scripts/utils/show_enh_score.sh -->
 <!-- These results are from the code before refactoring  -->
+
+# RESULTS
+## Environments
+- date: `Mon Aug  7 09:48:51 UTC 2023`
+- python version: `3.7.4 (default, Aug 13 2019, 20:35:49)  [GCC 7.3.0]`
+- espnet version: `espnet 202304`
+- pytorch version: `pytorch 1.10.1+cu111`
+- Git hash: `277ec3c33d2ca7f47d9d31c84e4dae54ce017bd7`
+  - Commit date: `Wed Aug 10 13:32:09 2022 -0400`
+- run.sh
+  - min_or_max=min
+  - sample_rate=16k
+  - mode=multi
+
+
+## enh_train_enh_tfgridnet_waspaa2023_raw
+
+config: ./conf/tuning/train_enh_tfgridnet_waspaa2023.yaml
+
+|dataset|STOI|SAR|SDR|SIR|SI_SNR|
+|---|---|---|---|---|---|
+|enhanced_tt_spatialized_anechoic_multich_min_16k|99.65|27.45|26.98|38.43|27.02|
+|enhanced_tt_spatialized_reverb_multich_min_16k|98.13|18.81|18.39|29.47|18.12|
```

Comment:
Is the model trained on anechoic or reverberant data?

Reply:
The model was trained on multi conditions (both anechoic and reverberant data). `mode=multi` indicates this point